### PR TITLE
[test] test whether the RCP supports tx/rx 154 frames of all formats

### PR DESCRIPTION
**Pull request by zhanglongxia**
This commit add a command `--frame-format` to the `cp-caps` to test whether the RCP supports sending and receiving 802.15.4 frames of all formats.

### CODE REVIEW

File: tools/cp-caps/README.md [hunk 1]
```diff
@@ -54,26 +54,28 @@ Show help info.
...
   -c, --csl            test whether the RCP supports CSL transmitter
-  -d, --diag-commands  test whether the RCP supports all diag commands
   -l, --link-metrics   test whether the RCP supports link metrics
+  -d, --diag-commands  test whether the RCP supports all diag commands
   -p, --data-poll      test whether the RCP supports data poll
-  -t, --throughput     test the Thread network 1-hop throughput
+  -t, --throughput     test Thread network 1-hop throughput
   -v, --version        output version
   -D, --debug          output debug information
+  --frame-format       test whether the RCP supports 802.15.4 frames of all formats
```
Comment (superwhd) [inline tools/cp-caps/README.md:70]: nit: Maybe
- group this line with other test cases
- introduce a short form argument (e.g. -f)

**Review by superwhd — APPROVED** [review]
LGTM with a nit

File: tools/cp-caps/rcp_caps_test.py [hunk 2]
```diff
@@ -552,12 +673,13 @@ def __connect_reference_device(self) -> OTCI:
 
         return node
 
-    def __output_format_string(self, name: str, value: str):
-        prefix = '{0:-<58}'.format('{} '.format(name))
+    def __output_format_string(self, name: str, value: str, align_length: int = DEFAULT_FORMAT_ALIGN_LENGTH):
+        name = name + ' '
+        prefix = name.ljust(align_length, '-')
```
Comment (Irving-cl) [inline tools/cp-caps/rcp_caps_test.py:678]:
```suggestion
        prefix = (name + ' ').ljust(align_length, '-')
```
This is simpler?

**Review by Irving-cl — APPROVED** [review]
LGTM 👍 

Some small suggestions.

**Review by jwhui — APPROVED** [review]

**Pull request merged**